specs/version_feature_forcast.md

```diff
@@ -24,10 +24,11 @@ Per-major version, what features would we like to have added? Note: **bug fixes
 
 ### v1.1
 
-- Autofilling Templates during character creation
+- Autofilling spells and weapons.
 - An enhanced wizard with more steps to guide people through (or a Skip button which creates an empty char)
 - Better Profile Image management/options
 - Planning for DM tools (not implementation)
+- Status modifiers for players (see: #645)
```

Comment:
I'm still not 100% sold on this. I feel like this needs a lot of design and orchestration before we begin implementation.